### ISSUE TITLE
wrong local variable "frontend_priv_ip_configuration_name" condition in locals.tf

### ISSUE THREAD

**Issue by peepeepopapapeepeepo** [claranet/terraform-azurerm-app-gateway#4]
locals.tf
``` hcl
locals {
  . . .
  frontend_ip_configuration_name      = var.frontend_ip_configuration_name != "" ? var.frontend_ip_configuration_name : join("-", [local.default_name, "frontipconfig"])
  frontend_priv_ip_configuration_name = var.frontend_priv_ip_configuration_name != "" ? var.frontend_ip_configuration_name : join("-", [local.default_name, "frontipconfig-priv"])
  . . .
}
```
should change return of `true` condition of  `frontend_priv_ip_configuration_name` 
from `var.frontend_ip_configuration_name` 
to `var.frontend_priv_ip_configuration_name`

like this
``` hcl
locals {
  . . .
  frontend_ip_configuration_name      = var.frontend_ip_configuration_name != "" ? var.frontend_ip_configuration_name : join("-", [local.default_name, "frontipconfig"])
  frontend_priv_ip_configuration_name = var.frontend_priv_ip_configuration_name != "" ? var.frontend_priv_ip_configuration_name : join("-", [local.default_name, "frontipconfig-priv"])
  . . .
}
```
Otherwise when assigning `frontend_priv_ip_configuration_name` myself, it will return "DuplicateResourceName" as follows

`Original Error: Code="InvalidRequestFormat" Message="Cannot parse the request." Details=[{"code":"DuplicateResourceName","message":"Resource /subscriptions//resourceGroups//providers/Microsoft.Network/applicationGateways/ has two child resources with the same name (agw-xxxxxx-az-asse-dev-001-frontpublicip)."}`

**Comment by Shr3ps** [comment]
Hello, thanks for this issue, we have identified this internally, this should be fixed in the next release.

**Comment by Shr3ps** [comment]
Hello, did you try latest version?

**Comment by Shr3ps** [comment]
This should be fixed by version v5.0.0 (https://github.com/claranet/terraform-azurerm-app-gateway/blob/master/CHANGELOG.md#v500---2021-10-04)
